scheduler/google/cloud/scheduler_v1/gapic/cloud_scheduler_client.py

```diff
@@ -98,13 +98,6 @@ def location_path(cls, project, location):
             location=location,
         )
 
-    @classmethod
```

Comment:
`project_path` being removed may be problematic given that this library is GA

Reply:
This method shouldn't have been added, and is getting removed as a side effect of moving to proto annotations. No samples (generated or handwritten) used this method. 

Impact to customers should be minimal, but will still note as breaking in the next release.